src/app/log-tool/visualize/visualize-menu/graph-basics/graph-basics.component.ts

```diff
@@ -32,6 +34,8 @@ export class GraphBasicsComponent implements OnInit {
         }
       }
     });
+    this.markerTypes = ["lines+markers", "lines", "markers"];
```

Comment:
for markerTypes do a type of {label: string, value: string} and capitalize the label for display.

Reply:
this.markerTypes = [{label: "Lines & Markers", value: "lines+markers"}]